src/test/resources/fixtures/codeStyle/multiple-return-points.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+trait A
+{
+    public function test()
+    {
+        if ($this instanceof \stdClass) {
+            return false;
+        }
+        return true;
+    }
+}
+
+class Test_A
+{
+    public function <warning descr="Method with multiple return points.">customMethod</warning>()
+    {
+
+        if ($this instanceof TestA) {
+            return false;
+        }
+
+        return true;
+    }
+}
+
+class Test_B
+{
+    public function customMethod()
```

Comment:
Almost, a method returning a lambda with a single return statement.
Returns in lambdas should not be counted.